content/200-concepts/150-data-platform/03-projects/800-upgrading-plans.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Upgrading plans'
+metaTitle: 'Upgrading plans'
+metaDescription: 'You can upgrade to a paid plan to increase the Prisma Data Proxy usage and the number of team members and environments that you can add to your project.'
+---
+
+<TopBlock>
+
+Each project you create in the Prisma Data Platform starts by running on a free plan (`Early Adopter` or `Free`).
+
+You can upgrade your project to a paid plan to increase the Prisma Data Proxy usage and the number of team members and environments that you can add to your project.
```

Comment:
I think "increase the proxy usage" could be phrased differently. Here, we are trying to document the reasons why someone would upgrade a project.

Taking a stab to frame it likes this, don't mind the exact wording:

You may want to upgrade your project for several reasons:
* you want to have more collaborators than the free (or EA) plan allows, or you need specific roles for your collaborators (viewer, editor, etc.) -> oppt to link to roles
* you want to have more environments than your current plan allows
* you have exceeded or are concerned about exceeding the proxy usage quota in your free plan, you want to benefit from unbound proxy usage 
* if you are on the EA plan, additionally, upgrading to the Scale plan, you can benefit from removing the concurrency limitations on the data proxy 


This still feels murky, so it could use some iteration. Happy to discuss further or iterate together.

Reply:
Edited your text a bit and it's now part of the intro on this page.